gravitee-apim-gateway/gravitee-apim-gateway-reactor/src/main/java/io/gravitee/gateway/jupiter/reactor/v4/subscription/context/SubscriptionRequest.java

```diff
@@ -86,57 +93,64 @@ public String transactionId() {
 
     @Override
     public String uri() {
-        throw new IllegalStateException();
+        return "";
```

Comment:
I understand why we do this, I suppose this is used here and there but won't this lead of to issue at some point ?

Reply:
Yeah I know... let's discuss it if we can find a proper way.